pkg/plugin/interface.go

```diff
@@ -75,13 +76,26 @@ type Interface interface {
 
 	// GetSourceURL returns the URL where the plugin came from and where updates to it will be located.
 	GetSourceURL() string
+
+	// GetOrder returns the order in which the plugin is run. Plugins will wait until all other plugins with lower order are done before being launched.
+	GetOrder() int
+}
+
+// Definition defines a plugin's features, method of launch, and other
+// metadata about it.
+type Definition struct {
```

Comment:
I'm confused about this diff. I see you adding a `Definition` type but I get the impression this was already there but an invalid diff. The issue is that I just can't see the corresponding removal of this type from the previous commit (e.g. normally with invalid diffs you see it added in one place and removed from the original; here we just see this new type)

Do you know why it may be showing like this?